CMakeLists.txt

```diff
@@ -5,6 +5,17 @@ else()
     project("mbed TLS" C)
 endif()
 
+include(CMakePackageConfigHelpers)
+
+set(CMAKE_EXPORT_DIR "lib/cmake/mbedtls")
+
+file(READ "${CMAKE_SOURCE_DIR}/include/mbedtls/version.h" MBEDTLS_VERSION_H)
+string(REGEX REPLACE ".*#define[\t ]+MBEDTLS_VERSION_MAJOR[\t ]+([0-9]+).*" "\\1" MBEDTLS_VERSION_MAJOR "${MBEDTLS_VERSION_H}")
+string(REGEX REPLACE ".*#define[\t ]+MBEDTLS_VERSION_MINOR[\t ]+([0-9]+).*" "\\1" MBEDTLS_VERSION_MINOR "${MBEDTLS_VERSION_H}")
+string(REGEX REPLACE ".*#define[\t ]+MBEDTLS_VERSION_PATCH[\t ]+([0-9]+).*" "\\1" MBEDTLS_VERSION_PATCH "${MBEDTLS_VERSION_H}")
+set(MBEDTLS_VERSION "${MBEDTLS_VERSION_MAJOR}.${MBEDTLS_VERSION_MINOR}.${MBEDTLS_VERSION_PATCH}")
```

Comment:
I didn't know about scripts/bump_version.sh, and I get your point about making another PR to isolate the version number change, but isn't it a bit overkill to demand a new PR to be created? The only modification that would be required is modifying bump_version.h such that it no longer modifies the cmake scripts, because it is now no longer necessary. As I understand it, config.h is the canonical reference for configuration options, version numbers, etc, so parsing it from the cmake script ensures it is always in sync. This also has the advantage of avoiding modifications in multiple places every time the version number is bumped.

Reply:
I'm for removing a place where the version number needs to be updated, if it can be done easily. But `bump_version.sh` must be updated accordingly.